operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -969,15 +969,18 @@ message ProxyConfig {
   // Sets the number of successive failed probes before indicating readiness failure.
   uint32 readinessFailureThreshold = 22;
 
+  // Sets the number of seconds after which the probe times out
+	uint32 readinessTimeoutSeconds = 23;
```

Comment:
you can not do this, please use 40.

Reply:
done, can you help fix those tests?
I can't figure where I need to update the tests that fails